docs/.markdownlintignore

```diff
@@ -1 +1,10 @@
-_site/
+.docusaurus
```

Comment:
Only need directories in here -- markdownlint automatically skips individual files.

Reply:
Got it